force-app/main/default/lwc/tableWithAccounts/tableWithAccounts.js

```diff
@@ -71,19 +73,27 @@ export default class TableWithAccounts extends LightningElement {
 
             })
             .catch(error => {
-                this.dispatchEvent(
-                    new ShowToastEvent({
-                        title: 'Error',
-                        message: 'Failed to retrieve accounts',
-                        variant: 'error'
-                    })
-                );
+                this.showToast('Success', 'Failed to retrieve accounts', 'error');
             })
             .finally(() => {
                 this.isLoading = false;
             });
     }
 
+    checkingOrg(idAccount) {
```

Comment:
which org? checking for what? name of the function should describe us what it does. At least check for what should be in the name I would say